src/nvme/mi.h

```diff
@@ -1020,6 +1254,81 @@ static inline int nvme_mi_mi_config_set_mctp_mtu(nvme_mi_ep_t ep, __u8 port,
 	return nvme_mi_mi_config_set(ep, dw0, mtu);
 }
 
+
+/**
+ * nvme_mi_mi_config_get_async_event - get configuration: Asynchronous Event
+ * @ep: endpoint for MI communication
+ * @aeelver: Asynchronous Event Enable List Version Number
+ * @list: AE Supported list header and list contents
+ * @list_num_bytes: number of bytes in the list header and contents buffer.
+ * This will be populated with returned size of list and contents if successful.
+ *
+ * Performs a MI Configuration Get, to query the current enable Asynchronous
+ * Events.  On success, populates @aeelver and the @list with current info,
+ *
+ * Return: The nvme command status if a response was received (see
+ * &enum nvme_status_field) or -1 with errno set otherwise..
+ */
+int nvme_mi_mi_config_get_async_event(nvme_mi_ep_t ep,
+				__u8 *aeelver,
+				struct nvme_mi_aem_supported_list *list,
+				size_t *list_num_bytes);
+
+/**
+ * nvme_mi_mi_config_set_async_event - set configuration: Asynchronous Event
+ * @ep: endpoint for MI communication
+ * @envfa: Enable SR-IOV Virtual Functions AE
+ * @empfa: Enable SR-IOV Physical Functions AE
+ * @encfa: Enable PCI Functions AE.
+ * @aemd: AEM Delay Interval (for Sync only)
+ * @aerd: AEM Retry Delay (for Sync only; time in 100s of ms)
+ * @enable_list: nvme_mi_aem_enable_listucture containing header and items
+ * of events to be enabled or disabled.  This is taken as a delta change
+ * from the current configuration.
+ * @enable_list_length: Length of the enable_list in bytes including header and data.
+ * Meant to catch overrun issues.
+ * @occ_list: Pointer to populate with the occurrence list (header and data)
+ * @occ_list_num_bytes: Total length of provided occ_list buffer in bytes.  Will be
+ * updated with received size if successful
```

Comment:
Minor inconsistency in naming here: the `enable_list_length` vs. `occ_list_num_bytes`.

A `_length` typically represents a number of elements, but it's a size (you mention "in bytes" in the comment). `_num_bytes` properly describes the latter, but `_size` is more conventional if properly commented.

I'd suggest `_size` for both, if we're working with sizes in bytes.